gridt/src/app/home/home.page.scss

```diff
@@ -5,4 +5,14 @@
 .leader-container {
     display: flex;
     justify-content: space-between;
-}
\ No newline at end of file
+}
+
+// The width of cards on the home page should match the width of the screen for small enough screens. Issue #60
+@media screen and (max-width: 600px) {
+  ion-card{
+    display: flex;
+    flex-direction: column;
+    width: 100% !important;
+    margin: 0 !important;
```

Comment:
Why are you using !important here? 

(Generally speaking this is seen as bad practice if it is not *absolutely* necessary.)